packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -629,14 +629,14 @@ describe('sqlite', () => {
 
     await expect(result).rejects.toMatchInlineSnapshot(`
 
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        ⚠️ We found changes that cannot be executed:
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ⚠️ We found changes that cannot be executed:
```

Comment:
Are we sure these are enough spaces yet? :trollface:

Reply:
To the moon 🙈